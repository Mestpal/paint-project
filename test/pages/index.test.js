
import Vue from 'vue'
import Vuetify from 'vuetify'
import Vuex from 'vuex'
import { shallowMount, createLocalVue } from '@vue/test-utils'

import index from '@/pages/index'

Vue.use(Vuetify)
Vue.use(Vuex)
const localVue = createLocalVue()

describe('index.vue page', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(index, {
      localVue
    })
  })

  it('reders the color selector component correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
