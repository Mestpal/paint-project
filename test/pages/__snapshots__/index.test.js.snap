// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue page reders the color selector component correctly 1`] = `
<v-layout-stub tag="div" justify-center="" align-center="">
  <v-container-stub tag="div" fluid="true">
    <v-row-stub tag="div" dense="true">
      <customcanvas-stub></customcanvas-stub>
      <v-col-stub cols="4" tag="div">
        <redoundone-stub></redoundone-stub>
        <colorselector-stub></colorselector-stub>
        <lineweightselector-stub></lineweightselector-stub>
      </v-col-stub>
    </v-row-stub>
  </v-container-stub>
</v-layout-stub>
`;
